### DISCUSSION TITLE
directory 구조에 관하여

### DISCUSSION THREAD

**Discussion by ljw0096** [prgrms-fe-devcourse/FEDC2_TimeDivider_Root#62] [1. 비개발]
1. Page를 디렉토리로 관리하기 

page - `index.js`, `style.js`

2. shared에 관하여

- shared는 컴포넌트에만 적용되어야 한다 (shared && component) 둘 사이만 연관이 있게 구분해야 한다
- utils까지 넣어서 정말 공통으로 사용되는 모든것이 들어가야 한다

**Comment by dahye1013** [comment]
1번 page를 디렉토리로 관리하는거 좋은 것 같아요!! 
스타일드 컴포넌트만 style.js로 빼서요!

2번은 어떤게 더 좋은지 고민 좀 더 해봐야 할 것 같네요 👀 
아래 의견 달아주세요!

**Comment by codeisneverodd** [comment]
src directory 내에서 이루어져야할 것 같습니다!


![image-20220617143335142](https://tva1.sinaimg.cn/large/e6c9d24egy1h3b6jzcnasj20ko0ppgn3.jpg)

**Reply by codeisneverodd** [reply]
1. pages 내부에 각 페이지별 폴더유지
2. 해당 page에서만 사용되는 component는 해당 페이지 디렉토리 내에 components 폴더에 유지
3. shared 내에는 공용으로 사용되는 것들만 유지
4. assets는 보류

**Reply by dahye1013** [reply]
stories 디렉토리도 shared에 넣는게 좋을까요?
